### DISCUSSION TITLE
custom database

### DISCUSSION THREAD

**Discussion by bionesiacourse** [hariszaf/pema#17] [General]
Hi Hariszaf. Can PEMA use a custom database to assign taxa. If yes what would be the format?. Thanks.Aji

**Comment by cpavloud** [comment]
Hi @bionesiacourse !
Yes, PEMA can use a custom reference database. 
But you will need 

a) to train the classifier that will do the taxonomy assignment (the [CREST classifier](https://hariszaf.github.io/pema_documentation/training_crest_classifier/) if you have 16/18S rRNA/ITS data or the [RDPClassifier](https://hariszaf.github.io/pema_documentation/training_rdpclassifier/) if you have COI data) and

b) to create a "custom_ref_db" folder containing a sequence alignment file (with a .fasta extension) and a taxonomy file (with a .nds extension) (in the CREST case) or a sequence file (with a .fasta extension) and a taxonomy file (with a .tsv extension) (in the RDPClassifier case).
